src/index.js

```diff
@@ -0,0 +1,90 @@
+import updateTaskState from './taskState';
+import {
+  addTodo, editTodo, deleteTodo, clearFinishedTasks,
+} from './task';
+import { saveTodoList, getTodoList } from './functions';
+import './style.css';
+
+const todo = [];
+const todoListContainer = document.querySelector('.todo-list');
+const clearButton = document.querySelector('.clear');
+const enterButton = document.querySelector('.enter');
+const todoFormInput = document.querySelector('.todo-form-input');
```

Comment:
I noticed you are using a camel case, that's nice but here for this word, "todo" you are not I think it is a little confusing for other people.

Reply:
nice observation @AlexRS90 . I guess I always viewed `todo` as one word instead of `toDo` as 2 different words 🤣.

Reply:
Yes, for example for is confusing because in Spanish "todo" means "all" so when I was reading I was a little confused until I understand the real meaning! To Do list 😆

Reply:
🤣 that's totally understandable @AlexRS90 . I guess I just learnt a new Spanish word: **todo** 😄